proto/kvrpcpb.proto

```diff
@@ -562,3 +563,12 @@ message ReadIndexResponse{
     uint64 read_index = 2;
 }
 
+message GetCommittedIndexAndTsRequest {
+    Context context = 1;
```

Comment:
Need to be `repeated Context context = 1` because one SQL can access many regions.

Reply:
Maybe we also need a new field `ts_required`. TiKV can leave `ts` as 0 in response if `ts_required` is false.